packages/element/src/react.js

```diff
@@ -179,18 +185,18 @@ export { Suspense };
  * @return {Array} The concatenated value.
  */
 export function concatChildren( ...childrenArguments ) {
-	return childrenArguments.reduce( ( memo, children, i ) => {
+	return childrenArguments.reduce( ( memoized, children, i ) => {
```

Comment:
`memo` was a poor choice of name to begin with, and was largely inherited from [old Underscore.js documentation](https://underscorejs.org/#reduce). There's a bit of inconsistency in how this argument is named elsewhere in the codebase, but "memoized" is not one of them, nor reflective of what the value is.

I might suggest `result` instead.

Reply:
Indeed, `result` is a better name for an accumulator. Thx.